app/top-page.module.css

```
@@ -0,0 +1,354 @@
+  margin-top: 10px;
+}
+.rentalSite,
+.informationSite {
+  width: 250px;              /* ★同じ幅にする */
+  padding: 18px 0;           /* 高さを揃える */
+  text-align: center;        /* 文字を中央揃え */
+  background-color: #1f71e4; /* 青 */
+  color: #ffffff;
+  border-radius: 8px;        /* 少し丸く */
+  font-size: 1.3rem;
+  font-family: Yu Mincho;
```

Stylelint (17.9.0)

[error] Expected quotes around "Yu Mincho" (font-family-name-quotes)

(font-family-name-quotes)